src/test/java/com/ui_init_setup/practiceproject/pages/SwagLabsLoginPage.java

```diff
@@ -35,6 +39,7 @@ protected void load() {
     @Override
     protected void isLoaded() throws Error {
```

Comment:
it would be preferable to assert the actual login form is present; since if the browser navigated to the correct page, but the form wasn't for whatever reason rendered - the test would still fail